specs/gloas/beacon-chain.md

```diff
@@ -823,7 +909,7 @@ def get_builder_withdrawals(
 ```python
 def get_expected_withdrawals(
     state: BeaconState,
-) -> Tuple[Sequence[Withdrawal], uint64, uint64, uint64]:
+) -> Tuple[Sequence[Withdrawal], uint64, uint64, uint64, uint64]:
```

Comment:
At this stage the return object of this function probably needs to be a named type